docs/content/policy-reference.md

```diff
@@ -357,6 +357,12 @@ complex types.
 | <span class="opa-keep-it-together">``output := trim_space(string)``</span> | ``output`` is a ``string`` representing ``string`` with all leading and trailing white space removed.|
 | <span class="opa-keep-it-together">``output := upper(string)``</span> | ``output`` is ``string`` after converting to upper case |
 
+### Numbers
+
+| Built-in | Description |
+| ------- |-------------|
+| <span class="opa-keep-it-together">``output := numbers.range(a, b)``</span> | ``output`` is the range of integer numbers between ``a`` and ``b`` (inclusive). If ``a`` == ``b`` then ``output`` == ``[a]``. If ``a`` < ``b`` the range is in ascending order. If ``a`` > ``b`` the range is in descending order. |
+
```

Comment:
Nit: `output = [a]`.

We already have a `Numbers` section in the Built-in function docs. Let's either combine or rename this.

Reply:
Merged this into the existing one. Forgot we had it.